gtfslib/dao.py

```diff
@@ -354,8 +357,7 @@ def _page_query(self, query_factory, item_feed_id_column, item_id_column, ids, b
         for feed_id, item_ids in group_pairs(ids, batch_size):
             query = query_factory()
             query = query.filter((item_feed_id_column == feed_id) & (item_id_column.in_(item_ids)))
-            batch = query.all()
-            for item in batch:
+            for item in query:
```

Comment:
Is there an impact on performance for this (I'm thinking about pre-fetching mode?)